cronjobs/search-tester/run_search_tester.py

```diff
@@ -64,6 +65,65 @@ def add_api_results(search: TestSearch, results: List[dict]):
             result.details = api_result['vehicleCollateral']
         search.results.append(result)
 
+def parse_results(batch_searches, rows, lower):
+    """Return batch_searches with all parsed rows information."""
+    row_keys = ['SEARCH_TYPE', 'TIME', 'CRITERIA', 'MATCH_TYPE', 'RESULT', 'DOCUMENT_ID', 'ID']
+    if lower:
+        row_keys = [k.lower() for k in row_keys]
+    for row in rows:
+        search_type = row[row_keys[0]]
+        time = row[row_keys[1]]
+
+        if time not in batch_searches[search_type]:
+            # add new search to batch
+            batch_searches[search_type][time] = {
+                'criteria': row[row_keys[2]],
+                'exact_matches': [],
+                'similar_matches': []
+            }
+
+        if row[row_keys[3]] == TestSearchResult.MatchType.EXACT.value:
+            # add exact match to search
+            batch_searches[search_type][time]['exact_matches'].append({
+                'result': row[row_keys[4]],
+                'doc_id': row[row_keys[5]],
+                'index': row[row_keys[6]]
+            })
+        else:
+            # add similar match to search
+            batch_searches[search_type][time]['similar_matches'].append({
+                'result': row[row_keys[4]],
+                'doc_id': row[row_keys[5]],
+                'index': row[row_keys[6]]
+            })
+    return batch_searches
+
+
+def get_batch_searches_csv(app: Flask, batch_searches: dict) -> dict:
+    """Get batches from csv."""
+    # read csv + prep for search tests
+    filename = app.config['FILE_NAME']
+    with open(f'csvs/{filename}', 'r') as csvfile:
```

Comment:
Does this need a try/catch?

Reply:
The fn is called within a try catch so I think it's okay. It's a temp job so I don't think we need to worry about it too much